docs/docs/cmd/spp/model/model-remove.mdx

```diff
@@ -0,0 +1,53 @@
+import Global from '/docs/cmd/_global.mdx';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# spp model remove
+
+Deletes a document understanding model
+
+## Usage
+
+```sh
+m365 spp model remove [options]
+```
+
+## Options
+
+```md definition-list
+`-u, --siteUrl <siteUrl>`
+: The URL of the content center site.
+
+`-i, --id [id]`
+: The unique ID of the model to delete. Specify either `id` or `title` but not both.
+
+`-t, --title [title]`
+: The display name (case-sensitive) of the model to remove. Specify either `id` or `title` but not both.
+
+`-f --force`
+: Don't prompt for confirming removing the model.
+```
+
+<Global />
+
+## Remarks
+
+Note that this model will be removed from all libraries before it can be deleted.
```

Comment:
Weird wording, shouldn't it be `should be removed`? Maybe it's worth to put this in a note admonition.